test/library/run-options.test.js

```diff
@@ -98,6 +98,31 @@ describe('Newman run options', function () {
         });
     });
 
+    describe('bail modifiers', function () {
+        it('should skip collection run in case of error when folder is specified', function (done) {
+            newman.run({
+                collection: 'test/fixtures/run/failed-request.json',
+                bail: ['folder']
```

Comment:
For this test to make sense, the folder option should also be specified.

Reply:
This basically just checks for error and whether or not the bail option affects it, specifying folder option will not change the core test behavior.